### DISCUSSION TITLE
Add support for Voltronic inverters via BT connection ?

### DISCUSSION THREAD

**Discussion by saviothecnic** [fl4p/batmon-ha#165] [General]
I tried this distribution and it was fantastic to have my Daly BMS on. Do you also have the question "card lovelace" code to get the graphics as in the example? Do you also intend to add support for Voltronic inverters via BT connection?

**Comment by fl4p** [comment]
it is possible, ~~here'a an esphome implementation: https://github.com/syssi/esphome-votronic~~

**Comment by saviothecnic** [comment]
I'm talking about Voltronic inverters specifically the King UNO
Here we are talking about Votronic which is another system with a similar name
but totally different

**Comment by fl4p** [comment]
ah ok.

what about  this https://github.com/dominikamann/InfiniSolar-Voltronic-Home-Assistant-Integration ?

**Reply by saviothecnic** [reply]
Thanks friend
But I was looking for something that spoke to voltronic inverters
with the wireless BlueTooth connection
How does your program with the Daly BMS
I thought you could also add this inverter to your program
By the way for the DalyBMS you have the lovelace to add to the HA dashboard
If you know how to talk to JK BMS maybe you can add
also BT communication with Neey Balancer
